src/app/(dashboard)/residents/create/page.tsx

```
@@ -175,99 +196,108 @@
+
+          {/* Pre-filled notification */}
+          {isPreFilled && suggestedName && (
+            <div className="mt-4 rounded-md bg-blue-50 p-3 dark:bg-blue-900/20">
+              <div className="flex">
+                <div className="flex-shrink-0">
+                  <svg
+                    className="h-5 w-5 text-blue-400"
+                    fill="none"
+                    stroke="currentColor"
+                    viewBox="0 0 24 24"
+                  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)